packaging/installer/dependencies/centos.sh

```diff
@@ -106,7 +105,22 @@ validate_tree_centos() {
 
   echo >&2 " > CentOS Version: $(os_version) ..."
 
-  if [[ $(os_version) =~ ^8(\..*)?$ ]]; then
+  if [[ $(os_version) =~ ^9(\..*)?$ ]]; then
+    package_manager=dnf
+    echo >&2 " > Checking for config-manager ..."
+    if ! dnf config-manager --help &> /dev/null; then
+      if prompt "config-manager not found, shall I install it?"; then
+        dnf ${opts} install 'dnf-command(config-manager)'
```

Comment:
Shouldn't we just install `dnf-plugins-core`?

Reply:
It’s generally considered better to explicitly call for the particular command we need like this, because it means we don’t care what specific package provides it.

Reply:
I would better add installation of the whole package at preparation step but ok.